crates/agentgateway/src/telemetry/log.rs

```diff
@@ -699,6 +699,85 @@ impl Drop for DropOnLog {
 					.and_then(|l| l.response.output_tokens)
 					.map(Into::into),
 			),
+			// OpenTelemetry Gen AI Semantic Conventions v1.37.0
+			(
+				"gen_ai.operation.name",
+				log.llm_request.as_ref().map(|_| "chat".into()),
+			),
+			(
+				"gen_ai.provider.name",
+				log.llm_request.as_ref().map(|l| display(&l.provider)),
+			),
+			(
+				"gen_ai.system",
```

Comment:
I don't see `gen_ai.system` in the list (https://opentelemetry.io/docs/specs/semconv/gen-ai/gen-ai-spans/), and its redudant with `gen_ai.provider.name`